src/main/java/com/jfrog/ide/common/tree/FileTreeNode.java

```diff
@@ -43,4 +46,10 @@ public void addDependency(ApplicableIssueNode issue) {
             topSeverity = issue.getSeverity();
         }
     }
+
+    public void sortChildren() {
+        if (CollectionUtils.isNotEmpty(children)) {
+            children.sort((treeNode1, treeNode2) -> ((DependencyNode) treeNode2).getTopSeverity().ordinal() - ((DependencyNode) treeNode1).getTopSeverity().ordinal());
```

Comment:
FileTreeNode's children are not necessarily DependencyNodes. They may be ApplicableIssueNodes, and other kinds of nodes might be added in the future.